ts/screens/wallet/payment/PaymentOutcomeCodeMessage.tsx

```diff
@@ -64,6 +64,9 @@ const successFooter = (onClose: () => void) => (
 const PaymentOutcomeCodeMessage: React.FC<Props> = (props: Props) => {
   const outcomeCode = props.outcomeCode.outcomeCode.fold(undefined, oC => oC);
 
+  // FIXME: this CTA will point to an external site, yet to be defined
+  const onLearnMore = () => null;
```

Comment:
@Undermaken the final URL is https://io.italia.it/faq/#pagamenti

Reply:
Updated in [01c15ac](https://github.com/pagopa/io-app/pull/3057/commits/01c15acd8a56958e36a7da56cfde21d8918861f7)
https://user-images.githubusercontent.com/11773070/163563996-c16425b2-1605-4fb7-a2f0-407bdf158600.mp4